Source/monster.cpp

```diff
@@ -1998,9 +1998,6 @@ bool IsTileSafe(const Monster &monster, Point position)
 	return true;
 }
 
-/**
- * @brief Check that the given tile is not currently blocked
- */
 bool IsTileAvailable(Point position)

```

Comment:
This is not the same function as `IsTileAvailable(Monster, Point)`. This function checks if a given position is free of monsters or players, then calls `IsTileWalkable(Point)` to check that the position is also free of objects and is a dungeon floor tile. It's declared here and not the header as this function isn't exposed to other cpp files.

Short story, this comment needs to be restored.